web-app/HeimScripts/heatmap/run.R

```diff
@@ -176,3 +189,85 @@ add.subset.label <- function(df,label){
   }
   return(df)
 }
+
+computeDendrogram <- function(zScoreMatrix, distanceMeassure, linkageMethod) {
```

Comment:
Fixed. It uses Z-scores now. They are recalculated - refactoring of this code to avoid recalculating -scores will come (but not in this PR).